src/prisma/generator/templates/actions.py.jinja

```diff
@@ -159,7 +159,7 @@ class {{ model.name }}Actions(Generic[{{ ModelType }}]):
         -------
         ```py
         # create a {{ model.name }} record from just the required fields
-        {{ model.name.lower() }} = {{ maybe_await }}{{ model.name }}.prisma().create(
+        {{ model.name | to_snake | lower }} = {{ maybe_await }}{{ model.name }}.prisma().create(
```

Comment:
note: I plan on changing how this works so that instead of doing `model.name.lower()` everywhere which makes changes like your one much more cumbersome.

In a separate PR I'll change this to `model.instance_name`.